src/async-syntax-highlighter.js

```diff
@@ -12,7 +12,7 @@ export default (options) => {
     static languages = [];
 
     static preload() {
-      ReactAsyncHighlighter.loadAstGenerator();
+      return ReactAsyncHighlighter.loadAstGenerator();
```

Comment:
The snapshots on master actually don't have syntax highlighting. This was caused by not returning the astGenerator promise in the preload method